spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/actions/TestRewriteDataFilesAction.java

```diff
@@ -300,7 +303,8 @@ public void testBinPackAfterPartitionChange() {
                 Integer.toString(averageFileSize(table) + 1000))
             .option(
                 RewriteDataFiles.TARGET_FILE_SIZE_BYTES,
-                Integer.toString(averageFileSize(table) + 1001))
+                // Increase max file size for V3 to account for additional row lineage fields
+                Integer.toString(averageFileSize(table) + (formatVersion >= 3 ? 11000 : 1001)))
```

Comment:
Curious if we come up with the `11000` number after a few tries?

Reply:
Yeah this was trial and error to find the amount that would make it so that on output of files for this particular test scenario we achieve the expected results consistently.